src/domain/user.js

```diff
@@ -213,11 +213,13 @@ export default class User {
     if (key !== undefined && value !== undefined) {
       query.where = {
         profile: {
-          [key]: value
+          [key]: {
+            contains: value,
+            mode: 'insensitive'
+          }
```

Comment:
Since there are no unit tests in this app, please manually test every route that relies on this function before merging the PR.

Reply:
manually tested every routes and one isn't working. Currently working on it